docs/src/advanced/arguments.md

```diff
@@ -16,6 +16,8 @@ Discordeno comes with the most useful command arguments already built for you. T
 - `member` When you want a member object and the user can provide a member id, @mention or their username/nickname.
 - `number` When you want a number.
 - `role` When you want a role object. The user can provide the role id, mention or role name.
+- `snowflake` When you just want a snowflake.
+- `..snowflakes` When you want to check if the mentioned ID is a valid snowflake.
```

Comment:
```suggestion
- `...snowflakes` When you want to check if the mentioned ID is a valid snowflake.
```

Reply:
I would also clarify that this lets you collect all the snowlakes not just one and that this should only be used as the last argument.

Reply:
That is irrelevant to the PR.